sample-queries.csv

```diff
@@ -4,6 +4,7 @@ Getting Started,GET,my photo,/v1.0/me/photo/$value,,https://developer.microsoft.
 Getting Started,GET,my mail,/v1.0/me/messages,,https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/user_list_messages,BOTH,,
 Getting Started,GET,all the items in my drive,/v1.0/me/drive/root/children,,https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/item_list_children,BOTH,,
 Getting Started,GET,items trending around me,/beta/me/insights/trending,,https://developer.microsoft.com/en-us/graph/docs/api-reference/beta/api/insights_list_trending,AAD,,
+Getting Started,GET,security alerts,/beta/security/alerts?$top=1,,https://developer.microsoft.com/en-us/graph/docs/api-reference/beta/api/alert_list,AAD,,
```

Comment:
The Getting Started section should not have queries that require admin consent. We'll need to remove this query from Getting Started.